packages/mobile/__mocks__/react-i18next.js

```diff
@@ -34,17 +34,28 @@ const renderNodes = (reactNodes) => {
   })
 }
 
-const useMock = [(k) => k, {}]
-useMock.t = (k) => k
+// This is useful to test that correct params were sent to i18n.
+// For example, in the TransferFeedItem tests we are checking that the title of the item matches a cached value.
+// Without this it's impossible to check if the used value is the cached one since it only prints the i18n key.
+const printParamInsteadOfKey = {
+  feedItemSentTitle: 'nameOrNumber',
+}
+const translationFunction = (key, params) => {
+  const paramToPrint = printParamInsteadOfKey[key]
+  return paramToPrint ? params[paramToPrint] || key : key
+}
+
+const useMock = [translationFunction, {}]
+useMock.t = translationFunction
```

Comment:
Nothing to do now, but it looks like we should revisit the decision of printing only the keys during unit test.
And print the actual EN strings.
What do you think?

Reply:
That does sound like it would be better, it's one of the things I considered but I think there were some issues around values that may change (dates maybe? I think there was something else). They can be definitely normalised though and it'd be nice to use real strings :)